RELEASE

```diff
@@ -0,0 +1,11 @@
+*******************************************************************************
+
+GEOSX: ................................, version 0.2.0
```

Comment:
why all the dots?

Reply:
what's wrong with dots? The more dots the better I always say.